modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cache/PlatformCache.java

```diff
@@ -744,16 +748,16 @@ public IgniteCache rawCache() {
 
                 case OP_INVOKE: {
                     Object key = reader.readObjectDetached();
-
-                    CacheEntryProcessor proc = platformCtx.createCacheEntryProcessor(reader.readObjectDetached(), 0);
+                    long ptr = reader.readLong();
+                    CacheEntryProcessor proc = platformCtx.createCacheEntryProcessor(reader.readObjectDetached(), ptr);
```

Comment:
Extract a private method to avoid code duplicates?

Reply:
I don't think it will improve readability, and it won't save LoC, so let's keep this as is.

Reply:
I don't think it will improve readability, and it won't save LoC, so let's keep this as is.